### PR TITLE
[change/fix] Wait until captive portal login response #265

### LINKED ISSUE #265: [bug] Current active session not shown after login
Depends on #259 for testing.

Once the user logs in and captive portal login is performed, the new open session should be shown in the UI but it's not, because sessions are loaded before the captive portal login.

Ideally we should load sessions only after the captive portal login has finished loading, we could even wait 1 extra second just to be sure.
Alternatively, we can reload sessions again, but we're already making a lot of requests and I'd avoid that.

### PR DESCRIPTION
**Pull request by nemesisdesign**
Wait until captive portal login response to perform actions like:

- clearing the loading overlay
- redirecting the user to payment flow
- loading user sessions

Fixes #265

### CODE REVIEW

File: browser-test/login.test.js [hunk 1]
```diff
@@ -47,5 +48,11 @@ describe("Selenium tests for <Login />", () => {
     await driver.wait(until.elementIsVisible(successToastDiv));
     await driver.wait(until.urlContains("status"), 5000);
     expect(await successToastDiv.getText()).toEqual("Login successful");
+
+    const activeSessionTr = await getElementByCss(
+      driver,
+      "table tr.active-session",
+    );
+    await driver.wait(until.elementIsVisible(activeSessionTr));
```
Comment (nemesisdesign) [inline browser-test/login.test.js:56]: unfortunately this doesn't fail for me without the fix, I was not able to replicate this issue in automated tests, only with manual testing :(

File: browser-test/utils.js [hunk 2]
```diff
@@ -34,6 +34,12 @@ export const getElementByXPath = async (driver, xpath) => {
   return el;
 };
 
+export const getElementByCss = async (driver, css) => {
```
Comment (nemesisdesign) [inline browser-test/utils.js:37]: @codesankalp let's use css instead of xpath which is more universally known

Comment (codesankalp) [reply browser-test/utils.js:37]: Added it in this https://github.com/openwisp/openwisp-wifi-login-pages/pull/284

File: browser-test/login.test.js (hunk 1 above)
Comment (codesankalp) [reply browser-test/login.test.js:56]: Yes, I also tried but sessions are showing in the browser test.

**Review by codesankalp — APPROVED** [review]

**Pull request merged**